dockers/base-cuda/Dockerfile

```diff
@@ -114,6 +114,10 @@ RUN \
     # install DeepSpeed
     pip install deepspeed==0.5.4
 
+RUN \
+    # install Bagua
+    pip install bagua-cuda102
```

Comment:
Do you have precompiled versions for different cuda-versions? We need to match this with the system cuda (defined in $CUDA_VERSION)

Reply:
Currently only three cuda-versions (10.2, 11.1 and 11.3) have pre-compiled packages. Otherwise, it will need to be installed from source code. We are discussing about adding for more cuda-versions. Will comment here if I have update. More info is here: https://tutorials.baguasys.com/installation#installation

Reply:
Late for this update. We can provide pre-compiled versions for CUDA 10.2, 11.1, 11.3 and 11.5(we already did so). They are built upon the [`pytorch/manylinux**`](https://hub.docker.com/search?q=pytorch%2Fmanylinux&type=image) images, it seems images for other cuda-versions are not actively maintained now. Would it be all right? @justusschock

Reply:
@justusschock